### PR TITLE
ID-2679 [Maintenance] Update ESLint config

### PR DESCRIPTION
**Pull request by tonytlwu**
Ref. https://weboo.atlassian.net/browse/ID-2679

### CODE REVIEW

**Review by squallstar — APPROVED** [review]

File: js/libs/core.js [hunk 1]
```diff
@@ -13,20 +12,20 @@ Fliplet.InteractiveMap = (function() {
     emit: function(eventName, data) {
       eventHub.$emit(eventName, data);
     },
-    component(componentName, component) {
```
Comment (squallstar) [inline js/libs/core.js:16]: Uhm, why is this not accepted anymore?

File: js/libs/interface.js [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-      new Sortable(el, binding.value || {})
+      new Sortable(el, binding.value || {});
     }
   }
 });
 
-const selector = '#interactive-map-app'
+const selector = '#interactive-map-app';
 
-const app = new Vue({
```
Comment (squallstar) [inline js/libs/interface.js:17]: Was this unused?

File: js/libs/core.js (hunk 1 above)
Comment (tonytlwu) [reply js/libs/core.js:16]: It is accepted, but I changed it to be consistent with the other functions defined above.

File: js/libs/interface.js (hunk 2 above)
Comment (tonytlwu) [reply js/libs/interface.js:17]: That's correct

**Pull request merged**